packages/nextjs/pages/api/impact/index.ts

```diff
@@ -1,13 +1,7 @@
-import csv from "csv-parser";
-import fs from "fs";
+import { getVectors } from "../vectors";
 import type { NextApiRequest, NextApiResponse } from "next";
-import path from "path";
-import { DataSet, ImpactVectors, RetroPGF3Results } from "~~/app/types/data";
-
-const dataFilePath = path.join(process.cwd(), "public", "data/RPGF3Results.csv");
-
-// Protocol Guild contains metrics for many different projects that are included individually
-const excludedProjects = ["Protocol Guild"];
```

Comment:
That was more for demo purposes. I think in the future when we add the filtering functionality we will probably use some master filters to clean the data set in a open and transparent way if needed, instead of hardcoding names of projects.

Reply:
Got it, thanks for the explanation